libs/input-mapping/src/Staging/AbstractDefinition.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\InputMapping\Staging;
+
+use Keboola\InputMapping\Exception\StagingException;
+
+abstract class AbstractDefinition
```

Comment:
Kdyz si se uz pustil do toho zavedeni `AbstractDefinition`, osobne bych to vzal jeste o krucek dal a udelal i `DefinitionInterface`, protoze lepsi typovat (a linkovat konstanty) na interface, nez abstract class.. ale je to detail.

Reply:
jo, souhlas, ale ne tady a ne teď :)

Reply:
ok